src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -147,7 +162,7 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
      * This will not subscribe to updates: remember to call subscribe() separately if
      * desired.
      */
-    public joinRoomSession(activeFoci: Focus[]): void {
+    public async joinRoomSession(activeFoci: Focus[], encryptMedia?: boolean): Promise<void> {
```

Comment:
Is this change related? Does something need to wait until the membership event is sent?

Reply:
This is a relict of the way to send the keys I used before - will revert